### ISSUE TITLE
Argument to pre-select which MFA device to use?

### ISSUE THREAD

**Issue by baranner** [grindsa/dkb-robo#53]
Hi there, 

thanks for the work so far! I was super exited to find your repo.

One thought though, i have two smartphones registered as second factor, so every time i try to pull my transactions for a finance statistics script i am writing i get this prompt:
```python

with DKBRobo(dkb_user="XXXXXXXX, dkb_password='XXXXXXXXXXX', tan_insert=False, legacy_login=False, debug=False) as dkb:
    acc = dkb.account_dic
```
Output:
```
Pick an authentication device from the below list:
[0] - DKB-App on iPhone
[1] - DKB-App on Fairphone FP4
```

could you maybe implement an argument to pre-select that device so one can avoid that user interaction? 

Or is there maybe one i overlooked? 

Cheers!

**Comment by grindsa** [comment]
Hi,

What you are asking for is currently not supported but and should not be difficult to implement.

I could introduce an "mfa_device" parameter with the following options.

- m - main device 
-  0|1 ... X - the number of the mfa_device from the list

Right now the devices are sorted randomly. I am thinking to change this and list the "main device" first and sort the others by enrollment date.

Thoughts?

**Comment by baranner** [comment]
Sounds awesome ❤️ that would be super convenient :)

**Comment by grindsa** [comment]
I just released [0.26](https://github.com/grindsa/dkb-robo/releases/tag/0.26) which allows mfa-device pre-selection.

```python
with DKBRobo(dkb_user="XXXXXXXX, dkb_password='XXXXXXXXXXX', tan_insert=False, legacy_login=False, mfa_device=1, debug=False) as dkb:
    acc = dkb.account_dic
```
mfa_device parameter can be set to "m" (use device form list) or the number of the device from the selection list:

```python
[1] - DKB-App on Google Pixel 6
[2] - DKB-App on Google Pixel Tablet
```

feel free to give it a try and let me know if it works...
